.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy app on server
+
+on: push
```

Comment:
Deployment should only run on push to master. Please change this once you're done testing.